### PR TITLE
Fix RabbitMQ documentation

### PR DESCRIPTION
**Pull request by XavierGeerinck**
The RabbitMQ documentation was lacking when RabbitMQ was used but messages arrived in the "Unacked" state rather than the "Ready" state, which can happens in frameworks such as Dapr.

This commit clarifies that this can be resolved through the HTTP interface and also points to the correct endpoint to utilize.

### CODE REVIEW

File: content/docs/2.0/scalers/rabbitmq-queue.md [hunk 1]
```diff
@@ -30,23 +30,27 @@ triggers:
...
     This is usually resolved from a `Secret V1` or a `ConfigMap V1` collections. `env` and `envFrom` are both supported.  
     The resolved host should follow a format like `amqp://guest:password@localhost:5672/vhost` or 
-    `https://guest:password@localhost:443/vhostname`
+    `http://guest:password@localhost:15672/vhostname`
 - `queueName`: Name of the queue to read message from. Required.
 - `queueLength`: Queue length target for HPA. Default is 20. Optional.
 - `protocol`: Protocol to be used for communication. Either `http` or `amqp`. It should correspond with the `host` value. 
 
-Note `host`/`hostFromEnv` has an optional vhost name after the host slash which will be used to scope API request.
+> **Note:** `host`/`hostFromEnv` has an optional vhost name after the host slash which will be used to scope API request.
```
Comment (tomkerkhove) [inline content/docs/2.0/scalers/rabbitmq-queue.md:38]: Let's align with other docs
```suggestion
> 💡 **Note:** `host`/`hostFromEnv` has an optional vhost name after the host slash which will be used to scope API request.
```

File: content/docs/2.0/scalers/rabbitmq-queue.md [hunk 2]
```diff
@@ -30,23 +30,27 @@ triggers:
...
+
+> **Important:** if you have unacknowledged messages and want to have these counted for the scaling to happen, make sure to utilize the `http` REST API interface which allows for these to be counted.
```
Comment (tomkerkhove) [inline content/docs/2.0/scalers/rabbitmq-queue.md:40]:
```suggestion
> ⚠ **Important:** if you have unacknowledged messages and want to have these counted for the scaling to happen, make sure to utilize the `http` REST API interface which allows for these to be counted.
```

**Review by tomkerkhove — APPROVED** [review]
LGTM with some suggestions, can you fix DCO please?

**Comment by XavierGeerinck** [comment]
> LGTM with some suggestions, can you fix DCO please?

Fixed DCO but messed up a bit in-between 😅 any way of signing commits automatically?

**Comment by zroubalik** [comment]
> > LGTM with some suggestions, can you fix DCO please?
> 
> Fixed DCO but messed up a bit in-between 😅 any way of signing commits automatically?

To sign commits, you can run

```
git commit -s
```

VSCode has a setting to sign all commits and I think that other IDEs have the same, if that's what you are asking for :)

**Review by zroubalik — COMMENTED** [review]
Looking good, could you please rebase your branch and force push (so it doesn't contain the Alibaba commit) thanks!

**Comment by XavierGeerinck** [comment]
> Looking good, could you please rebase your branch and force push (so it doesn't contain the Alibaba commit) thanks!

Awesome! Figured it out, thank for the help 😊 (not a pro in all this signing stuff haha)

**Review by zroubalik — APPROVED** [review]

**Pull request merged**